app/context/Deepgram.tsx

```diff
@@ -17,12 +17,13 @@ import {
   useState,
 } from "react";
 import { useToast } from "./Toast";
+import { useLocalStorage } from "../lib/hooks/useLocalStorage";
 
 type DeepgramContext = {
-  ttsOptions: SpeakSchema | undefined;
-  setTtsOptions: Dispatch<SetStateAction<SpeakSchema | undefined>>;
-  sttOptions: LiveSchema | undefined;
-  setSttOptions: Dispatch<SetStateAction<LiveSchema | undefined>>;
```

Comment:
No need to change these! This is not the error

Reply:
Let me check, I did this change because if you check the useLocalStorage then the `setValue` takes the SpeakSchema or LiveSchema and set it in the local storage, later this function is being passed as a values to context, If I don't change this then Typescript is giving me an error.

Reply:
The types that is was before were effectively the same as what you've replaced it with :) 

The error I attached before shows a parsing error in the useLocalStorage file

Reply:
Hey @lukeocodes  let me know if anything else is needed from my side here, kindly have a look at the above comment ☝️

Reply:
Yeh, the build is still failing :/  Does it `npm run build` locally? Or just `npm run dev` locally?

Reply:
Also we've pushed up some changes, could you look at the conflicts with main?

Reply:
![Screenshot 2024-04-12 164601](https://github.com/deepgram-devs/deepgram-conversational-demo/assets/22278438/59777b1c-8af4-46bf-a1c8-a7766817e0a9)

Let me rebase and re-check

Reply:
I have rebased, checked the local build and its building without any error